### ISSUE TITLE
python3.8

### ISSUE THREAD

**Issue by edumotya** [monocongo/cvdata#155]
Would you add python3.8 support?

**Comment by monocongo** [comment]
Yes, we should be able to add support for both 3.8 and 3.9, as I am not aware of any limitations. Thanks for the impetus, @edumotya !
